### ISSUE TITLE
Decimal value of currency beyond 2 points doesn't compute

### ISSUE THREAD

**Issue by Drafyr** [Zrips/CMI#4352]
When I try using the placeholder %cmi_iteminhand_worth% it displays the true value including all 3 decimal places of the item, but appends it with the currency term automatically. This causes my sellshop to not even register how much it's supposed to pay or even take the items saying "can't find player with this name". When I alternatively use the %cmi_worth_sell_{cmi_iteminhand_type}:{cmi_iteminhand_itemdata}% it works... But only to 2 decimal points.

```
####################
## 09.10 19:00:29 [Server] INFO Can't find player with this name! ##
####################
```

**Cmi Version (using`/cmi version`):** 8.7.8.1

**Server Type (Spigot/Paperspigot/etc):** PAPER

**Server Version (using `/ver`):** PAPER 213

**Comment by Zrips** [comment]
%cmi_iteminhand_worth% is formatted by economy plugin, if economy plugin supports only 2 decimals then it will only show 2 and if economy plugin adds currency symbol, then we will show that too. Entire formatting is left for economy plugin. You can always clear out any none numeric characters and convert into double value, if you need to get that.
For cmi_worth_sell placeholder, well that one is kind of a bug, it should have been formatted by economy plugin too, will fix that with next release.

**Comment by Drafyr** [comment]
I removed the denomination names from Gringotts "Emerald" and "Emeralds" but it isn't updating. The update you released fixed the formatting of the worth to show the decimal places properly, but now it appends it with "Emeralds" and I've checked the messages and config files and none of them have "Emeralds" showing up. It's showing the correct amount of decimal places, but again, it appends it with the economy "denomination" from Gringotts, that I left blank; but originally had set to "Emerald/Emeralds". Idk what to do. :<

**Comment by Drafyr** [comment]
Nvm! I hadn't uploaded the newest update that fixed it. Thank you!

**Comment by Drafyr** [comment]
OK so the "sellshop" works, but it automatically adds a space after the economy value to every instance of %cmi_worth_buy/sell%

I went into Gringotts and removed the names of the currency, but it still appends a space after all of the worth values of items listed in the worth file. Like, if I type %cmi_iteminhand_worth% (which is what made the sellshop work now after update).... It shows the proper values...

Long story short: I have the denominations names set as '' specifically, but it still appends a space after the values and so the command to buy has a space in it and therefore won't work. Possibly add a feature to toggle appending?
